### PR TITLE
Add docs for command line client for RPC 

### PR DESCRIPTION
**Pull request by s-martin**
Adding some docs how to use the CLI Client for RPC commands

Changes from #1857 are probably necessary as well

### CODE REVIEW

File: documentation/developers/cli-client.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Installation
+
+* Install prerequisites: `sudo apt-get install libczmq-dev`
+* Change to directory: `cd ~/RPi-Jukebox-RFID/src/cli_client`
+* Compile CLI client: `gcc pbc.c -o pbc -lzmq -Wall`
+
+## Usage
+
+* Get help info: `./pbc -h`
+* **TODO**
```
Comment (AlvinSchiller) [inline documentation/developers/cli-client.md:14]: Link the builders/rpc-commands.md  page?

File: documentation/developers/README.md [hunk 2]
```diff
@@ -11,3 +11,8 @@
 * [RFID Readers](./rfid)
 * [Feature Status](./status.md)
 * [Known Issues](./known-issues.md)
+
+## RPC
+
+* [LibZMQ](./libzmq.md)
+* [CLI Client for RPC](./cli-client.md)
```
Comment (AlvinSchiller) [inline documentation/developers/README.md:18]: If i get it right this is not necessarily only for developers.
Should the CI Client in the builders/Readme.md under "Advanced" ?

There is also the page builders/rpc-commands.md

File: documentation/developers/cli-client.md (hunk 1 above)
Comment (s-martin) [reply documentation/developers/cli-client.md:14]: Yes, good idea. Will do that

**Comment by AlvinSchiller** [comment]
Should the Cli-Client also be added to the [system.md](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/system.md), as from code level its one of the three jukebox main entries.

Also it could be linked from [concepts](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/develop/documentation/builders/concepts.md)?

File: documentation/developers/README.md (hunk 2 above)
Comment (s-martin) [reply documentation/developers/README.md:18]: You are probably right, builders should have the possibility to utilize everything.

**Review by pabera — APPROVED** [review]

**Pull request merged**